tests/e2e/configurer/config/constants.go

```diff
@@ -14,7 +14,7 @@ const (
 	// number of blocks it takes to vote for a single validator to vote for a proposal
 	PropVoteBlocks float32 = 1
 	// number of blocks used as a calculation buffer
-	PropBufferBlocks float32 = 8
+	PropBufferBlocks float32 = 30
```

Comment:
In the previous version of E2E, the block speed pre upgrade was normal, and the block speed post upgrade was accelerated. Now, block speed is accelerated both pre and post upgrade, so I increased the buffer accordingly.